pkgs/applications/office/timeline/default.nix

```diff
@@ -0,0 +1,90 @@
+{ lib
+, fetchurl
+, python3Packages
+, python3
```

Comment:
please use one or the other, but not both. Looks like python3 isn't actually used

Reply:
it is used in `${python3.interpreter} tools/execute-specs.py` in `checkPhase`

`${python3Packages.interpreter}` does not work:

> error: attribute 'interpreter' missing

what should i do?

Reply:
I would just use `python3`, and the package set can be reference through `python3.pkgs`

Reply:
was `python3Packages` deprecated?